scripts/globals/spells/diaga_iii.lua

```diff
@@ -15,52 +15,47 @@ end
 
 function onSpellCast(caster, target, spell)
 
-    --calculate raw damage
     local basedmg = caster:getSkillLevel(dsp.skill.ENFEEBLING_MAGIC) / 3
     local params = {}
     params.dmg = basedmg
     params.multiplier = 5
     params.skillType = dsp.skill.ENFEEBLING_MAGIC
     params.attribute = dsp.mod.INT
     params.hasMultipleTargetReduction = false
-
-    local dmg = calculateMagicDamage(caster, target, spell, params)
-
-    dmg = utils.clamp(dmg, 1, 60)
-
-    --get resist multiplier (1x if no resist)
-    local params = {}
-    params.diff = caster:getStat(dsp.mod.INT)-target:getStat(dsp.mod.INT)
+    params.diff = caster:getStat(dsp.mod.INT) - target:getStat(dsp.mod.INT)
     params.attribute = dsp.mod.INT
     params.skillType = dsp.skill.ENFEEBLING_MAGIC
     params.bonus = 1.0
+
+    -- calculate raw damage
+    local dmg = calculateMagicDamage(caster, target, spell, params)
+    -- softcaps at 60, should always do at least 1
+    dmg = utils.clamp(dmg, 1, 60)
+    -- get resist multiplier (1x if no resist)
     local resist = applyResistance(caster, target, spell, params)
-    --get the resisted damage
+    -- get the resisted damage
     dmg = dmg * resist
-    --add on bonuses (staff/day/weather/jas/mab/etc all go in this function)
+    -- add on bonuses (staff/day/weather/jas/mab/etc all go in this function)
     dmg = addBonuses(caster, spell, target, dmg)
-    --add in target adjustment
+    -- add in target adjustment
     dmg = adjustForTarget(target, dmg, spell:getElement())
-    --add in final adjustments including the actual damage dealt
+    -- add in final adjustments including the actual damage dealt
     local final = finalMagicAdjustments(caster, target, spell, dmg)
 
-    -- Calculate duration and bonus
+    -- calculate duration and bonus
     local duration = calculateDuration(120, spell:getSkillType(), spell:getSpellGroup(), caster, target)
     local dotBonus = caster:getMod(dsp.mod.DIA_DOT)  -- Dia Wand
 
-    -- Check for Bio.
+    -- check for Bio
     local bio = target:getStatusEffect(dsp.effect.BIO)
 
-    -- Do it!
-    if target:addStatusEffect(dsp.effect.DIA, 3 + dotBonus, 3, duration, 0, 15, 3) then
-        spell:setMsg(dsp.msg.basic.MAGIC_DMG)
-    else
-        spell:setMsg(dsp.msg.basic.MAGIC_NO_EFFECT)
-    end
+    -- do it!
+    target:addStatusEffect(dsp.effect.DIA, 3 + dotBonus, 3, duration, 0, 20, 3)
+    spell:setMsg(dsp.msg.basic.MAGIC_DMG)
 
-    -- Try to kill same tier Bio
-    if BIO_OVERWRITE == 1 and bio ~= nil then
-        if bio:getPower() <= 3 then
+    -- try to kill same tier Bio (non-default behavior)
+    if (BIO_OVERWRITE == 1 and bio ~= nil) then
```

Comment:
no parens

Reply:
will fix along with whatever else comes, I just picked one for consistency as some had some didn't.